health-services/resource-generator/src/main/java/org/egov/processor/service/ExcelParser.java

```diff
@@ -230,12 +239,46 @@ private void processSheets(PlanConfigurationRequest request, String fileStoreId,
 					processRowsForCensusRecords(request, excelWorkbookSheet,
 							fileStoreId, attributeNameVsDataTypeMap, boundaryCodeList, campaign.getCampaign().get(0).getHierarchyType());
 				} else if (request.getPlanConfiguration().getStatus().equals(config.getPlanConfigUpdatePlanEstimatesIntoOutputFileStatus())) {
+
+					// Create the map of boundary code to the facility assigned to that boundary.
+					Map<String, String> boundaryCodeToFacility = outputEstimationGenerationUtil.getBoundaryCodeToFacilityMap(excelWorkbook, request, fileStoreId);
+					Map<String, String> facilityToFixedPost = fetchFixedPostDetails(request, boundaryCodeToFacility);
```

Comment:
add comment and move call getBoundaryCodeToFacilityMap inside fetchFixedPostDetails function